openpype/plugins/publish/collect_audio.py

```diff
@@ -0,0 +1,83 @@
+import pyblish.api
+from pprint import pformat
+
+from openpype.client import (
+    get_last_version_by_subset_name,
+    get_representations,
+)
+from openpype.pipeline import (
+    legacy_io,
+    get_representation_path,
+)
+
+
+class CollectAudio(pyblish.api.InstancePlugin):
+    """ Collecting available audio subset to instance
+
+    """
+    label = "Collect Audio"
```

Comment:
Would this be more descriptive as `Collect Asset Audio` or `Collect Audio from Asset`?